utilities/data-provider/gsutil_sync.bat

```diff
@@ -0,0 +1,24 @@
+::  This file automates how data are sync'd to a
+::  Google Cloud Cloud Storage bucket using a
+::  pre-existing service account key.
+::
+::  Notes:
+::  ----------------------------------------------------
+::  presumes gsutil has already been installed and is
+::  available in the path.
+::  see gsutil docs for more information:
+::  https://cloud.google.com/storage/docs/gsutil_install
+
+::  authenticate gcloud for the service account
+::  note: this is the preferred method for authenticating gsutil
+::  see the following for more details:
+::  https://cloud.google.com/storage/docs/gsutil/commands/config#configuring-service-account-credentials
+call gcloud auth activate-service-account --key-file=service-account.json
```

Comment:
This line still references the nonexistent `service-account.json` file.

Maybe you c invoke `gcloud auth login` here instead, to establish their user credentials? I'm not 100% sure, but I believe that command will detect if the user is already logged in and skip reauthenticating them, unless you pass `--force`.

Reply:
Thanks, I updated this as well! I found that the `gcloud auth login` always will attempt to login, even if the session is already authenticated for the user. Either way, the command appears to halt until the browser session is ended or the command is cancelled, so I feel good about this working as is (granted someone has browser access). If this doesn't work, they could still use `rclone` as another workaround.